### PR TITLE
Router

### PR DESCRIPTION
**Pull request by vitali-yablonski**

### CODE REVIEW

File: src/pages/Details.vue [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            Movie,
+            Button
+        },
+        data: function () {
+            return {
+                backgroundImage: backgroundImage,
+            }
+        },
+        methods: {
+            ...mapActions(['fetchMovie']),
```
Comment (MariMax) [inline src/pages/Details.vue:37]: I'd make a const FETCH_MOVIE = 'fetchMovie', in this case you can use it in many places without worring to make typos

File: src/pages/Details.vue [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            onClickHome: function () {
+                this.$router.push({ name: 'home' })
```
Comment (MariMax) [inline src/pages/Details.vue:39]: sam here, you can assign routes names to the constants

File: src/pages/Details.vue [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            },
+        },
+        computed: {
+            ...mapState(['movie'])
```
Comment (MariMax) [inline src/pages/Details.vue:43]: same here

**Review by MariMax — APPROVED** [review]

File: src/pages/Details.vue (hunk 1 above)
Comment (vitali-yablonski) [reply src/pages/Details.vue:37]: ok, thanks

File: src/pages/Details.vue (hunk 2 above)
Comment (vitali-yablonski) [reply src/pages/Details.vue:39]: ok, thanks

File: src/pages/Details.vue (hunk 3 above)
Comment (vitali-yablonski) [reply src/pages/Details.vue:43]: ok, thanks

**Pull request merged**